backend/app/services/mongo/explore.py

```diff
@@ -2967,8 +2965,9 @@ async def compute_cloud_of_clusters(
             },
             {
                 "$project": {
-                    "task_id": 1,
-                    "session_id": 1,
+                    "task_id": str(1),
+                    "session_id": str(1),
+                    "user_id": str(1),
```

Comment:
Don't do that and instead change the type hint of pipeline
pipeline: List[Dict[str, object]]